server/meshmodel/kubernetes/v1.32.0-alpha.1/v1.0.0/relationships/edge-binding-firewall-kllzw.json

```diff
@@ -63,7 +67,10 @@
                   "id": null,
                   "kind": "NetworkPolicy",
                   "mutatorRef": [
-                    []
+                    ["configuration", "spec", "podSelector", "matchLabels"],
+                    ["configuration", "spec", "policyTypes","_"],
+                    ["configuration", "spec", "ingress", "_", "from", "_", "podSelector", "matchLabels"],
```

Comment:
This won't work there needs to be a single `_` in a set.

Reply:
ohh okay let me review this